css/templates/e-commerce/page.css

```diff
@@ -140,6 +140,10 @@ dd:after {
   margin-bottom: var(--space-1);
 }
 
+.ampstart-input {
+  width: 100%;
```

Comment:
Ensure input fields are full width on mobile. On desktop they are as wide as the column they're in.